web/template/team-remove-member.gotmpl

```diff
@@ -0,0 +1,50 @@
+{{ template "page" . }}
+
+{{ define "backlink" }}
+  <a class="govuk-back-link" href="{{ prefix (printf "/teams/%d" .Team.ID) }}">Back</a>
+{{ end }}
+
+{{ define "title" }}Remove users from team{{ end }}
+
+{{ define "main" }}
+  {{ template "error-summary" .Errors }}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl">Remove users from team</h1>
+
+      {{ if gt (len .Selected) 2 }}
+        <p class="govuk-body">
+          Are you sure you want to remove the following members from the <strong>{{ .Team.DisplayName }}</strong> team?
+        </p>
+        <ul class="govuk-list govuk-list--bullet">
+          {{ range .Selected }}
+            <li><strong>{{ .DisplayName }}</strong></li>
+          {{ end }}
+        </ul>
+      {{ else if gt (len $.Selected) 1 }}
+        <p class="govuk-body">
+          Are you sure you want to remove <strong>{{ (index .Selected 0).DisplayName }}</strong> and <strong>{{ (index .Selected 1).DisplayName }}</strong> from the <strong>{{ .Team.DisplayName }}</strong> team?
+        </p>
+      {{ else }}
+        <p class="govuk-body">
+          Are you sure you want to remove <strong>{{ (index .Selected 0).DisplayName }}</strong> from the <strong>{{ .Team.DisplayName }}</strong> team?
+        </p>
+      {{ end }}
+
+      <form class="form" action="" method="post">
+        {{ range .Selected }}
+          <input type="hidden" name="selected[]" value="{{ .ID }}" />
+        {{ end }}
+
+        <button type="submit"  class="govuk-button" data-module="govuk-button" name="confirm" value="confirm">
```

Comment:
Can you add `govuk-!-margin-right-1` for nicer spacing

Reply:
Much better 👍